### ISSUE TITLE
Add Repo input to config component

### ISSUE THREAD

**Issue by wei** [wei/socialify#68]
This way users can switch between repos without the need to go back to home page. 

This will also allow the config to be persisted when switching between repos.

**Comment by KemingHe** [comment]
Yes @wei, I was just going to post this as an issue, glad I read all the docs first, haha. Are you still looking for help on this one? Shouldn't take long for me to do. Just let me know.

**Comment by KemingHe** [comment]
Just realized this issue has been here since 2020, might as well call it a feature. 😄 JK

**Comment by wei** [comment]
> Yes @wei, I was just going to post this as an issue, glad I read all the docs first, haha. Are you still looking for help on this one? Shouldn't take long for me to do. Just let me know.

Yes @KemingHe 😁. That'd be amazing!

**Comment by wei** [comment]
@KemingHe assigning to you for now in case you're interested to implement this one.

**Comment by KemingHe** [comment]
Hi, I'm back to actively contributing to this, been a bit busy lately. All good now. Update you soon! @wei
